app/queries/topics_resolver.rb

```diff
@@ -63,6 +89,6 @@ def applicable_topics
       by_category.where(target_id: target_id)
     else
       by_category
-    end.order('topics.created_at DESC')
+    end.order("#{sort_criterion_string} #{sort_direction_string}")
```

Comment:
Since the `topics` table is joined to another, shouldn't this have a `topics.` prefix?

```suggestion
    end.order("topics.#{sort_criterion_string} #{sort_direction_string}")
```